lightning/src/chain/chaininterface.rs

```diff
@@ -35,15 +35,19 @@ pub trait BroadcasterInterface {
 	fn broadcast_transactions(&self, txs: &[&Transaction]);
 }
 
-/// An enum that represents the speed at which we want a transaction to confirm used for feerate
+/// An enum that represents the priority at which we want a transaction to confirm used for feerate
 /// estimation.
 #[derive(Clone, Copy, Debug, Hash, PartialEq, Eq)]
 pub enum ConfirmationTarget {
-	/// We are happy with this transaction confirming slowly when feerate drops some.
+	/// We are happy with a transaction eventually confirming, regardless of how long it may take,
+	/// using the absolute minimum feerate required to enter most node mempools across the network.
+	MempoolMinimum,
```

Comment:
Would it make sense to also use this new confirmation target in `OutboundV1Channel::new`?